c/article.amp.html

```diff
@@ -0,0 +1,382 @@
+<!doctype html>
```

Comment:
we have this assert for cid usage https://github.com/ampproject/amphtml/blob/master/src/service/cid-impl.js#L149